pages.ar/common/fastmod.md

```diff
@@ -0,0 +1,25 @@
+# fastmod
+
+> أداة للاستبدال الجزئي للنصوص في قاعدة الأكواد لديك.
+> التعبيرات الاعتيادية يعالجها قفص من بضاعة رست وهو regex.
+> لمزيد من العلومات: <https://github.com/facebookincubator/fastmod>.
+
+- استبدال التعبيرات الاعتيادية في كل ملفات المسار الحالي وأبنائه في الملفات غير المُتجاهلة بـ .ignore أو .gitignore
+
+`fastmod {{regex_pattern}} {{replacement}}`
```

Comment:
Please translate the placeholders for get this easier for non-English speakers. Like `path/to/directory` and `regex_pattern`.